Hide unneeded options for file/stdin inputs.

JWahle opened the pull request:
- When switching to file/stdin inputs, unneeded options (`target_level`, `enable_rate_adjust`, `adjust_period`) are hidden. When switching back, they are restored.
- `queuelimit` is changed to 1 when switching to file/stdin input and to 100 when switching to other inputs.
- Renamed option "resampler type" to "resampler_type" for consistency.

HEnquist commented:
I have changed the default queuelimit to 4 in 0.5.0 since the large value was mostly causing confusing problems. I haven't had time to try it yet, but I think there is a good chance that 4 also works well with stdin. We should check if 4 works well with the cdsp plugin.

JWahle commented:
A `queuelimit` of 4 should be OK for file/stdin inputs.
I removed the code that switches between `queuelimit` 1 and 100 and changed the default in the GUI to 4.
